elide-async/src/main/java/com/yahoo/elide/async/operation/JSONAPITableExportOperation.java

```diff
@@ -33,15 +34,19 @@
 public class JSONAPITableExportOperation extends TableExportOperation {
 
     public JSONAPITableExportOperation(TableExportFormatter formatter, AsyncExecutorService service,
-            AsyncAPI export, RequestScope scope) {
-        super(formatter, service, export, scope);
+            AsyncAPI export, RequestScope scope, ResultStorageEngine engine) {
+        super(formatter, service, export, scope, engine);
     }
 
     @Override
-    public RequestScope getRequestScope(TableExport export, User user, String apiVersion, DataStoreTransaction tx)
-            throws URISyntaxException {
+    public RequestScope getRequestScope(TableExport export, User user, String apiVersion, DataStoreTransaction tx) {
         UUID requestId = UUID.fromString(export.getRequestId());
-        URIBuilder uri = new URIBuilder(export.getQuery());
+        URIBuilder uri;
+        try {
+            uri = new URIBuilder(export.getQuery());
+        } catch (URISyntaxException e) {
+            throw new BadRequestException(e.getMessage());
```

Comment:
IllegalStateException would make more sense here.  I"m not sure this should ever happen (and there would be nothing the client can do to resolve it).

Reply:
if the query is malformed, then we can get this error which can be considered a BadRequest.